Modules/Core/Common/include/itkSmapsFileParser.h

```diff
@@ -277,8 +277,13 @@ class ITK_TEMPLATE_EXPORT MapFileParser
   GetMemoryUsage(const char * filter, const char * token = "Size");
 
 protected:
-  std::string m_MapFilePath{}; //< location of the last loaded Map file
-  TMapData    m_MapData{};     //< data of the loaded smap file
+  /** location of the last loaded Map file
```

Comment:
Does doxygen generate docs for protected member variables? If not, this change might not make much sense.